doc/api/toolkits/axes_grid.rst

```diff
@@ -0,0 +1,27 @@
+.. _axes_grid-api-index:
+
+####################################
+The Matplotlib axes_grid Toolkit API
+####################################
+
+:Release: |version|
+:Date: |today|
+
+.. currentmodule:: mpl_toolkits
+
+Axes Grid
+---------
+.. note::
+   
+   There is an older version of the AxesGrid toolkit, simply called ``axes_grid`` instead of
+   ``axes_grid1``.  the toolkit had a single namespace of axes_grid. This toolkit was broken
+   into the two modules below. For the documentation on the original ``axes_grid``
```

Comment:
When I teach colleagues about making good design decisions early on, I always use this as an example of what can happen when hasty decisions are made.